autopilot/stim/sound/sounds.py

```diff
@@ -1,18 +1,26 @@
-"""
-Classes to play sounds.
-
-Each sound inherits a base type depending on `prefs.get('AUDIOSERVER')`
-
-* `prefs.get('AUDIOSERVER') == 'jack'` : :class:`.Jack_Sound`
-* `prefs.get('AUDIOSERVER') == 'pyo'` : :class:`.Pyo_Sound`
-
-To avoid unnecessary dependencies, `Jack_Sound` is not defined if AUDIOSERVER is `'pyo'`
-and vice versa.
+"""This module defines classes to generate different sounds.
+
+These classes are currently implemented:
+* Tone : a sinuosoidal pure tone
+* Noise : a burst of white noise
+* File : read from a file
+* Speech
```

Comment:
hahahahaha i forgot these were in here. thank the lord for the plugin system so i can take all my lab-specific stuff out

Reply:
I hope we can keep "The Humble Sine Wave" in the documentation somewhere!